### PR TITLE
Use URL and require build ID instead of basing it on planet date stamp

### PR DESCRIPTION
**Pull request by iandees**
This makes the `cron.py` command accept a URL and run_id instead of inferring it from the planet date stamp.

Additionally:
* Add cost tags to resources that can be tagged
* Don't use `optimal` batch compute instances, because that uses r4 instances that are old as heck
* Optionally allow a small number of missing metatiles to pass a build
* Use a more recent, bigger, ARM-based Postgres instance type for RDS
* Write out a small sample of the tiles that will get re-enqueued when a pass fails

### CODE REVIEW

File: import/database.py [hunk 1]
```diff
@@ -157,6 +165,12 @@ def take_snapshot_and_shutdown(db, run_id):
         SkipFinalSnapshot=True,
     )
     waiter = rds.get_waiter('db_instance_deleted')
-    waiter.wait(DBInstanceIdentifier=instance_id)
+    waiter.wait(
+        DBInstanceIdentifier=instance_id,
+        WaiterConfig=dict(
```
Comment (rmarianski) [inline import/database.py:170]: :zap:

File: utils/setup.sh [hunk 2]
```diff
@@ -69,7 +69,7 @@ export AWS_DEFAULT_REGION="${REGION}"
 
 title "Creating S3 buckets"
 for func in tile-assets rawr-tiles missing-tiles meta-tiles; do
-   aws s3 mb "s3://${PREFIX}-${func}-${REGION}" --region "${REGION}" || error_exit
+   aws s3 mb "s3://${PREFIX}-${func}-${REGION}" --region "${REGION}" || echo "Failed to create s3://${PREFIX}-${func}-${REGION}"
```
Comment (rmarianski) [inline utils/setup.sh:72]: do we not want to fail here on error?

**Review by rmarianski — APPROVED** [review]

File: utils/setup.sh (hunk 2 above)
Comment (iandees) [reply utils/setup.sh:72]: Yea, that's a good question. I don't remember making this change. Let me figure out where it came from.

Comment (iandees) [reply utils/setup.sh]: Put it back!

**Pull request merged**